src/utils/vaultMath.ts

```diff
@@ -42,3 +43,8 @@ export const computeToLock = (
 
   return floorDivideBy(receiveMargin, priceRate).value;
 };
+
```

Comment:
the return value could use some explaining
```suggestion
/**
 * @returns tuple of [value of difference, boolean of whether it's negative
 */
```